packages/web-components/src/components/va-table/va-table-inner/va-table-inner.tsx

```diff
@@ -47,6 +47,22 @@ export class VaTableInner {
    */
   @Prop() stacked?: boolean = false;
 
+  @Prop() sortdir?: string = null;
+
+  @Prop() sortindex?: number = null;
```

Comment:
If `sortdir` and `sortindex` are public properties, can we add documentation comments for them to explain how they work? Would it also be valuable to add a story that demonstrates them?

Reply:
@jamigibbs - good call - they don't need to be props, I have handled this using data- attrs and State in a refactor